docker-compose.yaml

```diff
@@ -11,3 +11,4 @@ services:
     privileged: true
     volumes:
       - /dev:/dev
+      - /run/udev:/run/udev
```

Comment:
Is this the correct directory for udev? i'm less experienced so i don't know everything but i believe udev rules live in /etc/udev but i could be wrong

Reply:
i think u are correct, /etc/udev is where the udev rules files should live. 
however i think the /run/udev directory may serves a different purpose. will look more into it